pkg/controller/route/route_controller.go

```diff
@@ -94,7 +96,8 @@ func New(routes cloudprovider.Routes, kubeClient clientset.Interface, nodeInform
 	return rc
 }
 
-func (rc *RouteController) Run(stopCh <-chan struct{}, syncPeriod time.Duration) {
+// Run starts the route controller until the provided stop channel is closed
```

Comment:
Quibbling but my confidence that the channel will actually stop the controller is not high. (I don't think we've ever actually  tested that it works). Might be better to say something like "starts the controller reconciling routes against node changes every five minutes".